### PR TITLE
DOC-2162: Improve documentation for `noneditable` options and multi-root editing

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2162, Improve structure, content, and presentation of `noneditable` and multi-root documentation.

Changes:
* initial outline with non-editable related options extracted to non-editable-content-options.adoc
* updated references in ie-template-creation.adoc and added admonition referencing back to non-editable-content-options.adoc

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/
 
 ### Unreleased
 
+### TBA
```
Comment (FarzadHayat) [inline changelog.md:9]: add date before merging

**Review by kemister85 — APPROVED** [review]

**Comment by kemister85** [comment]
@FarzadHayat Nice work on this PR 🥳

**Review by MitchC1999 — APPROVED** [review]
Nice work 👍

File: changelog.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/
...
```
Comment (FarzadHayat) [inline changelog.md:9]:
```suggestion
### 2024-02-13
```

**Pull request merged**